src/hdf5/tropics_2ioda.py

```diff
@@ -35,11 +35,16 @@
 obsValName = iconv.OvalName()
 
 # globals
-TROPICS01_WMO_sat_ID = 691
+TROPICS01_WMO_sat_ID = 933  # 691  did this ID get updated by WMO
```

Comment:
@BenjaminTJohnson did you see this?  the WMO ID for the TROPICS pathfinder apparently changed I got this from WMO OSCAR page when I was looking up the TROPICS-06 and -07

are these IDs in your coefficient files?